js/src/collapse.js

```diff
@@ -298,6 +309,31 @@ const Collapse = (($) => {
       }
     }
 
+    _getShowClass() {
+      const tabClass = this._element.classList
+
+      // Detect flex for inline style
+      let useFlex = $(this._element).css('display').indexOf('flex') !== -1
+
+      // Create a wrapper to hide our flex detection
```

Comment:
I can use that for the line 316 yes, but not for the detection below because getComputedStyle return the current applied style and that's not what I want here. 
I want to know if they are a class which contain a flex display